<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::Application::TimingInfoView</class>
 <widget class="QWidget" name="Elpida::Application::TimingInfoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>137</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="lblVirtualOverhead">
     <property name="toolTip">
      <string>The average time it takes to call a virtual class function.</string>
     </property>
     <property name="text">
      <string>VCall overhead:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblNowOverhead">
     <property name="toolTip">
      <string>The average time it takes for timestamp calls to return.</string>
     </property>
     <property name="text">
      <string>now() overhead:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblNowOverheadValue">
     <property name="toolTip">
      <string>The average time it takes for timestamp calls to return.</string>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblVirtualOverheadValue">
     <property name="toolTip">
      <string>The average time it takes to call a virtual class function.</string>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblLoopOverhead">
     <property name="toolTip">
      <string>The average time it takes to perform a loop iteration.</string>
     </property>
     <property name="text">
      <string>Loop overhead:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblLoopOverheadValue">
     <property name="toolTip">
      <string>The average time it takes to perform a loop iteration.</string>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblIpsValue">
     <property name="toolTip">
      <string>The &quot;Bogus Instructions Per Second&quot; for a single core. It is how many empty simple loops can the processor can perform in a second. It is calculated for Processing Unit (0).</string>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblIps">
     <property name="toolTip">
      <string>The &amp;quot;Bogus Instructions Per Second&amp;quot; for a single core. It is how many empty simple loops can the processor can perform in a second. It is calculated for Processing Unit (0).</string>
     </property>
     <property name="text">
      <string>B. Ips:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
